projects/step-frontend/src/lib/modules/execution/components/execution-list/execution-list.component.html

```diff
@@ -155,5 +155,20 @@
       </th>
       <td mat-cell *matCellDef></td>
     </ng-container>
+    <ng-container *stepTablePaginatorContent>
+      <button
+        type="button"
+        mat-icon-button
+        [color]="calculateCounts() ? 'primary' : undefined"
+        [matTooltip]="
+          calculateCounts()
+            ? 'Click to not load total number of executions to improve the performance'
+            : 'Click to load total number of executions'
```

Comment:
```suggestion
            : 'Click to load the total number of executions. Note: this may slow down performance for large numbers of executions'
```